### ISSUE TITLE
One of two fields required

### ISSUE THREAD

**Issue by whoopah** [formvalidation/formvalidation#887]
I have a two phone fields on my form - phone and phone2. Only one of them is required. The way I would like it to work is check each field and if both are empty then display a custom message under each. If either one, or both, has data then go ahead and validate the field with data for correctness and mark the other one as not required anymore. If/when the user clicks submit again, check both fields again and if either or both has status message, remove the message start validation on both all over again.

Basically, one of the two fields needs to have data and that data needs to be valid. I don't think the fields being phone numbers is important, it should probably work with any two fields.

Is there a way to do this already?
